README.md

```diff
@@ -330,7 +330,7 @@ $ drive push -piped path
 ```
 
 + Note:
-  * In relation to [#107](https://github.com/odeke-em/drive/issues/107) and numerous other issues related to confusion about clashing paths, drive will abort on trying to deal with clashing names. To turn off this safety, pass in flag `--ignore-name-clashes`.
+  * In relation to [#107](https://github.com/odeke-em/drive/issues/107) and numerous other issues related to confusion about clashing paths, drive will abort on trying to deal with clashing names. To turn off this safety, pass in flag `--ignore-name-clashes`. As an alternative you can use `--fix-clashes` option when running `pull` or `push` commands and drive will try to rename clashing files so adding unique suffixes.
```

Comment:
Awesome. Could we make the message stand out more as in mention before `--ignore-name-clashes` the new functionality to auto add unique suffixes/counts. Your new feature is way more useful and needed than `ignore-name-clashes`. I'll send some info when I get off this mobile and onto a computer.